ql/test/library-tests/semmle/go/dataflow/PromotedFields/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+func source() string {
+	return "hello world"
+}
+
+func sink(s string) {}
+
+type Inner struct {
+	field string
+}
+
+type Middle struct {
+	Inner
```

Comment:
We should make one of the embedded fields have a pointer type, to check that we're dealing with that correctly.
```suggestion
	*Inner
```

Reply:
Currently we do not support this: https://github.com/github/codeql-go-team/issues/270